src/main/java/ch/njol/skript/expressions/ExprTernary.java

```diff
@@ -41,16 +41,16 @@
 
 @Name("Ternary")
 @Description("A shorthand expression for returning something based on a condition.")
-@Examples({"set {points} to 500 if {admin::%player's uuid%} is set else 100"})
+@Examples({"set {points} to 500 if {admin::%player's uuid%} is set else 100", "set {points} to (player's name = \"Notch\") ? 500 : 100"})
 @Since("2.2-dev36")
 @SuppressWarnings("null")
 public class ExprTernary<T> extends SimpleExpression<T> {
```

Comment:
Preferably (or at least based on the current conventions), there should be no tabs and/or spaces on an empty line. Although it doesn’t matter *too* much, but if you are making other changes you could probably remove them.